Source/Core/VideoBackends/OGL/FramebufferManager.cpp

```diff
@@ -239,156 +364,47 @@ FramebufferManager::FramebufferManager(int targetWidth, int targetHeight, int ms
     // the values of each sample.
     if (m_EFBLayers > 1)
```

Comment:
This could probably be `sampler = m_EFBLayers > 1 ? GLSL_SHADER_MSAA_MULTILAYER : GLSL_SHADER_MSAA_SINGLELAYER;`.

Could also turn this into a constant, e.g. `const bool multilayer = m_EFBLayers > 1;` (I realize it wasn't this way in the original code).